src/main/java/seedu/savenus/storage/savings/JsonAdaptedSavings.java

```diff
@@ -49,6 +48,10 @@ public Savings toModelType() throws IllegalValueException {
         if (!Money.isValidMoney(this.savingsAmount)) {
             throw new IllegalValueException(Savings.MESSAGE_CONSTRAINTS);
         }
```

Comment:
This exception is thrown when trying to load the saving data because isvalidmoney only allows non-negative amounts but the withdrawals are saved as negative amounts, might want to reconsider this part!

Reply:
Reconsidered this implementation. Do review !